pkg/server/config.go

```diff
@@ -42,6 +61,36 @@ func (cfg *HandlerConfig) Validate() error {
 	return nil
 }
 
+// ValidateMappingConfig validates the handler config for mapping service after load.
+func (cfg *HandlerConfig) ValidateMappingConfig() error {
+	if err := cfg.Validate(); err != nil {
+		return err
```

Comment:
Let's follow up in the next PR, we can join each error and return at the end, the original Validate func also need to be updated. here is an example - https://github.com/abcxyz/access-on-demand/blob/490e6a99855f0faf87a416cf221f1240bec003d5/apis/v1alpha1/validation.go#L79

Reply:
Created #140